src/commands/exec.ts

```diff
@@ -383,7 +384,7 @@ Alternatively, running "architect --% exec -- ls" will prevent the PowerShell pa
       compose_args.push(arg);
     }
 
-    await DockerComposeUtils.dockerCompose(compose_args, { stdio: 'inherit' }, true);
+    await DockerComposeUtils.dockerCompose(compose_args, { stdio: 'inherit' }, use_tty);
```

Comment:
The default value for `flags.tty` is undefined. Doesn't this change the default behavior?

Reply:
If `flags.tty` isn't explicitly set by the user it defaults to `flags.tty = !!process.stdin.isTTY;` in the main `run` method before we get here, so at this point it'll always be a bool